xarray/plot/utils.py

```diff
@@ -28,20 +28,9 @@
 ROBUST_PERCENTILE = 2.0
 
 
-_registered = False
-
-
-def register_pandas_datetime_converter_if_needed():
-    # based on https://github.com/pandas-dev/pandas/pull/17710
-    global _registered
-    if not _registered:
-        pd.plotting.register_matplotlib_converters()
-        _registered = True
-
-
 def import_matplotlib_pyplot():
     """Import pyplot as register appropriate converters."""
-    register_pandas_datetime_converter_if_needed()
+    # TODO: This function doesn't do anything, remove it?
```

Comment:
This function doesn't do much anymore without the registration. It can probably be removed later once this PR has been tested a bit in the field.